[2025-04-28T06:54:28.646+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_locations scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-04-28T06:54:28.655+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_ETL_DAG.insert_locations scheduled__2025-03-10T00:00:00+00:00 [queued]>
[2025-04-28T06:54:28.655+0000] {taskinstance.py:1361} INFO - Starting attempt 4 of 5
[2025-04-28T06:54:28.669+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): insert_locations> on 2025-03-10 00:00:00+00:00
[2025-04-28T06:54:28.679+0000] {standard_task_runner.py:57} INFO - Started process 609 to run task
[2025-04-28T06:54:28.683+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'Airflow_ETL_DAG', 'insert_locations', 'scheduled__2025-03-10T00:00:00+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/airflow_etl.py', '--cfg-path', '/tmp/tmp6iz1ldmu']
[2025-04-28T06:54:28.686+0000] {standard_task_runner.py:85} INFO - Job 21: Subtask insert_locations
[2025-04-28T06:54:28.752+0000] {task_command.py:416} INFO - Running <TaskInstance: Airflow_ETL_DAG.insert_locations scheduled__2025-03-10T00:00:00+00:00 [running]> on host 77b690a9fde8
[2025-04-28T06:54:28.846+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Airflow_ETL_DAG' AIRFLOW_CTX_TASK_ID='insert_locations' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T00:00:00+00:00'
[2025-04-28T06:54:29.562+0000] {logging_mixin.py:151} INFO - Insertion des locations termin√©e.
[2025-04-28T06:54:29.565+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-04-28T06:54:29.575+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Airflow_ETL_DAG, task_id=insert_locations, execution_date=20250310T000000, start_date=20250428T065428, end_date=20250428T065429
[2025-04-28T06:54:29.619+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-04-28T06:54:29.645+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
